src/dotnet-roslyn-tools/Commands/PRTaggerCommand.cs

```diff
@@ -45,9 +45,7 @@ public async Task<int> InvokeAsync(InvocationContext context)
             var logger = context.SetupLogging();
             var settings = context.ParseResult.LoadSettings(logger);
 
-            if (string.IsNullOrEmpty(settings.GitHubToken) ||
-                string.IsNullOrEmpty(settings.DevDivAzureDevOpsToken) ||
-                string.IsNullOrEmpty(settings.DncEngAzureDevOpsToken))
+            if (string.IsNullOrEmpty(settings.GitHubToken))
```

Comment:
Lets retain the AzDO token check when not using interactive authentication
```suggestion
            var isMissingAzDOToken = string.IsNullOrEmpty(settings.DevDivAzureDevOpsToken) || string.IsNullOrEmpty(settings.DncEngAzureDevOpsToken);
            if (string.IsNullOrEmpty(settings.GitHubToken) ||
                (settings.IsCI && isMissingAzDOToken))
```